build.gradle

```diff
@@ -0,0 +1,48 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+
+}
+
+plugins {
+    id 'com.github.johnrengelman.shadow' version '2.0.4'
+    id 'java-library'
+}
+
+//This allows you to build a single shadowJar with the contents of all the
+// subprojects included in it
+dependencies {
+    compile subprojects
+}
+
+subprojects {
+    apply plugin: 'java-library'
+
+    group 'org.htsjdk'
+    version '0.0.1'
```

Comment:
Versioning with gradle will be also nice (for another issue). I am interested in this one at the moment, which looks nice also to release to github: https://github.com/ajoberstar/reckon

Reply:
opening #20